chef/cookbooks/crowbar/templates/default/crowbar-lbaas-evacuation.sh.erb

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+#
+# This script migrates all neutron lbaasv2 loadbalancers from the node
+# that is passed in via the command line argument.
+
+LOGFILE=/var/log/crowbar/node-upgrade.log
+UPGRADEDIR=/var/lib/crowbar/upgrade
+mkdir -p "`dirname "$LOGFILE"`"
+exec >>"$LOGFILE" 2>&1
+
+hostname=$1
+if [[ "$2" = "delete-ns" ]]; then
+    delete_ns="--delete_namespaces"
```

Comment:
Note to self: I either need to add the --norestart_source_agent flag here as well (or make that implicit in the lbaas tool)